docs/core/whats-new/dotnet-9/sdk.md

```diff
@@ -115,6 +115,12 @@ If you have feedback about the terminal logger, you can provide it in the [MSBui
 
 Starting in .NET 8, `dotnet restore` [audits NuGet package references for known vulnerabilities](../../tools/dotnet-restore.md#audit-for-security-vulnerabilities). In .NET 9, the default mode has changed from auditing only _direct_ package references to auditing both _direct_ and _transitive_ package references.
 
+## Faster NuGet dependency resolution for large repos
+
+The NuGet dependency resolver has been overhauled to improve performance and scalability for all `<PackageReference>` projects. Enabled by default, the new algorithm speeds up restore operations without compromising on functionality, strictly adhering to the core dependency resolution rules.
+
+If you encounter any issues, such as restore failures or unexpected package versions compared to previous releases, you can [revert to the legacy resolver](https://learn.microsoft.com/nuget/consume-packages/Package-References-in-Project-Files#nuget-dependency-resolver).
```

Comment:
This is great. We will have a known issues section in 6.12 release notes as well, do you think it's something we should link from here?

Reply:
maybe we update the central docs that this links to w/ that known issues link?